src/pkg/mcp/setup.go

```diff
@@ -226,220 +201,109 @@ func getClientConfigPath(homeDir, goos string, client MCPClient) (string, error)
 	return filepath.Join(basePath, clientInfo.configFile), nil
 }
 
-// getDefangMCPConfig returns the default MCP config for Defang
-func getDefangMCPConfig(client MCPClient) (*MCPServerConfig, error) {
-	currentPath, err := os.Executable()
-	if err != nil {
-		return nil, err
-	}
-
-	return &MCPServerConfig{
-		Command: currentPath,
-		Args:    []string{"mcp", "serve", "--client", string(client)},
-	}, nil
-}
-
-// getVSCodeDefangMCPConfig returns the default MCP config for Defang in VSCode format
-func getVSCodeDefangMCPConfig(client MCPClient) (*VSCodeMCPServerConfig, error) {
-	currentPath, err := os.Executable()
+func readConfig(configPath string) (map[string]any, error) {
+	config := make(map[string]any)
+	configBytes, err := os.ReadFile(configPath)
 	if err != nil {
-		return nil, err
+		if os.IsNotExist(err) {
+			return config, nil
+		}
+		return nil, fmt.Errorf("failed to read config file: %w", err)
 	}
 
-	return &VSCodeMCPServerConfig{
-		Type:    "stdio",
-		Command: currentPath,
-		Args:    []string{"mcp", "serve", "--client", string(client)},
-	}, nil
-}
-
-// getVSCodeServerConfig returns a map with the VSCode-specific MCP server config
-func getVSCodeServerConfig(client MCPClient) (*VSCodeMCPServerConfig, error) {
-	config, err := getVSCodeDefangMCPConfig(client)
-	if err != nil {
-		return nil, err
+	if strings.TrimSpace(string(configBytes)) == "" {
```

Comment:
Could you not check the bytes instead of using strings?
`len(bytes.TrimSpace(configBytes)) == 0`

Reply:
sure, I didn't know about `bytes.TrimSpace`

```suggestion
	if bytes.TrimSpace(configBytes) == "" {
```